### PR TITLE
Add tab bar with no links

### LINKED ISSUE #1850: Pan European Home. Tab bar (for logged users) with Overview, Manage Collaborators and Recent Activity
Image

### PR DESCRIPTION
**Pull request by persas**
Closes #1850. Navigation tab bar for registered users

### CODE REVIEW

**Comment by persas** [comment]
Regarding this issue. I am not sure where the links of the tab bar should point to. In the FRA case, we have a bunch of pages that are under the `pages/AssessmentHome/FraHome` folder.  Do we have to move this to a different location and point to the same pages? Should it be different pages for panEu ?

**Comment by sorja** [comment]
Can we have a screenshot or video?

**Comment by sorja** [comment]
`import { useSections } from '../FraHome/hooks/useSections'`

Can be either moved one level higher to `AssessmentHome` or you can create similar one under `PanEuropeanHome` and add some Placeholder sections

**Comment by persas** [comment]
image

**Comment by minotogna** [comment]
> Regarding this issue. I am not sure where the links of the tab bar should point to. In the FRA case, we have a bunch of pages that are under the `pages/AssessmentHome/FraHome` folder.  Do we have to move this to a different location and point to the same pages? Should it be different pages for panEu ? 

They should point to similar path , like for Fra .
Can u import the components from fra home and then we'll see where to put them ?
So that we enable navigation in this pr .
Would that work ?

**Comment by persas** [comment]
Links working with FRA components
image

**Comment by persas** [comment]
https://drive.google.com/file/d/1rocmYBIQ9am6mgi6624Dw_rXXTt42f3z/view?usp=share_link

File: src/client/pages/AssessmentHome/PanEuropeanHome/PanEuropeanHome.tsx [hunk 1]
```diff
@@ -1,15 +1,58 @@
...
+
+import classNames from 'classnames'
+
+import { AssessmentHomeRouteNames, ClientRoutes } from '@meta/app'
+import { Areas } from '@meta/area'
+
+import { useCountryIso } from '@client/hooks'
+import User from '@client/pages/User'
+
+// eslint-disable-next-line import/no-unresolved
```
Comment (minotogna) [inline src/client/pages/AssessmentHome/PanEuropeanHome/PanEuropeanHome.tsx]: can we remove this disable ?

File: src/client/pages/AssessmentHome/PanEuropeanHome/hooks/useSections.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const Placeholder: React.FC = () => {
+  return <div />
+}
+
+export const useSections = (): Array<Section> => {
+  const user = useUser()
+  const countryIso = useCountryIso()
+  const cycle = useCycle()
+
+  const sections: Array<Section> = [{ name: SectionNames.overview, component: Placeholder }]
```
Comment (minotogna) [inline src/client/pages/AssessmentHome/PanEuropeanHome/hooks/useSections.tsx]: the overview component must be created and includes:
Screenshot 2022-12-13 at 12 24 06

**Review by minotogna — CHANGES_REQUESTED** [review]

**Comment by minotogna** [comment]
Also, can you check the why style of the links doesn't match fra ?

**Comment by minotogna** [comment]
> `import { useSections } from '../FraHome/hooks/useSections'`
> 
> 
> 
> Can be either moved one level higher to `AssessmentHome` or you can create similar one under `PanEuropeanHome` and add some Placeholder sections

I would leave things there . We'll refactor afterwards

**Comment by persas** [comment]
https://user-images.githubusercontent.com/23346350/207572354-92b88446-b7e9-440c-a54a-fdfd2d12e9c6.mov

**Review by minotogna — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**